skan/src/ui/Board.scala

```diff
@@ -136,19 +109,16 @@ object Board:
             )
       else Spans.nostyle("TODOs")
 
-    frame.renderStatefulWidget(
-      ListWidget(
-        items = todoItems,
-        block = Some(
-          BlockWidget(
-            borders = Borders.ALL,
-            title = Some(todoBorderTitle)
-          )
-        ),
-        highlightStyle = Style(bg = Some(Color.Gray), fg = Some(Color.Black))
-      ),
-      horizontalChunks(0)
-    )(state.todoState)
+    val todoBoard = Widget: (area, buf) =>
+      BlockWidget(
+        borders = Borders.ALL,
+        title = Some(todoBorderTitle)
+      )(
+        ListWidget(
+          items = todoItems.map(item => toListItem(item, area.width)),
```

Comment:
This is another interesting one  @oyvindberg that maybe I'm just using the `area.width` when I should instead be using something else, but I want it to know how much space to put between `the title                        the date`  for one to be left aligned and the other right aligned on the same line. In order to achieve that I need to know how wide the widget will be. So I can wrap it in a `Widget`, but idn, sort of stinks that I need to do that. Maybe it's fine, idn. Thinking about it, with the cursor example it'd be fine to get it after I put the layout together, but here it's sort of a chicken and the egg problem.